contextily/plotting.py

```diff
@@ -181,11 +181,16 @@ def add_attribution(ax, att=ATTRIBUTION, font_size=ATTRIBUTION_SIZE):
     """
     minX, maxX = ax.get_xlim()
     minY, maxY = ax.get_ylim()
-    ax.text(
+    txt = ax.text(
         minX + (maxX - minX) * 0.005,
         minY + (maxY - minY) * 0.005,
         att,
         size=font_size,
         path_effects=[patheffects.withStroke(linewidth=2, foreground="w")],
+        wrap=True,
     )
+    # hack to have the text wrapped in the ax extent, for some explanation see
+    # https://stackoverflow.com/questions/48079364/wrapping-text-not-working-in-matplotlib
+    wrap_width = ax.get_window_extent().width * 0.99
```

Comment:
I changed this from 0.9 to 0.99 so we have the same margin left and right (0.005 at both sides -> remaining text width of 0.99)